sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -38,6 +38,9 @@ struct ECSConfig {
 
     #[serde(rename = "TaskIAMRoleEnabledForNetworkHost")]
     task_iam_role_enabled_for_network_host: bool,
+
+    #[serde(rename = "SELinuxCapable")]
```

Comment:
Let's add a link to the [`types.go`](https://github.com/aws/amazon-ecs-agent/blob/a250409cf5eb4ad84a7b889023f1e4d2e274b7ab/agent/config/types.go) package that define the field names in the doc string (so it appears in the readme). The JSON structure is defined [there](https://github.com/aws/amazon-ecs-agent/blob/a250409cf5eb4ad84a7b889023f1e4d2e274b7ab/agent/config/types.go) and is authoritative for the "pass through" names we are seeing here.

Reply:
Done